backend/src/apiserver/visualization/roc_curve.py

```diff
@@ -34,7 +36,7 @@
 # trueclass
 # true_score_column
 
-if "is_generated" is not in variables or variables["is_generated"] is False:
+if not variables.get("is_generated"):
```

Comment:
I think that we should revert this change and move it to a separate PR. What are your thoughts?

Reply:
That change has been in #1878 for 4 days (a long time for a syntax error) and was not acted on so I bundled it it here